requirements.txt

```diff
@@ -0,0 +1,6 @@
+click==6.6
+Flask==0.11.1
+itsdangerous==0.24
+Jinja2==2.8
+MarkupSafe==0.23
+Werkzeug==0.11.11
```

Comment:
if we need just `requirements.txt`  for the beanstalk deploy we should either remove `setup.py` or make it use the same requirements.txt file in the `install_requires`